src/status_im/chat/handlers/input.cljs

```diff
@@ -229,11 +231,12 @@
                                          :sending-in-progress? false}])
           (react-comp/dismiss-keyboard!))
         (dispatch [::request-command-data
-                   {:content   content
-                    :chat-id   chat-id
-                    :jail-id   (or bot chat-id)
-                    :data-type :preview
-                    :after     #(dispatch [::send-message % chat-id])}])))))
+                   {:content         content
+                    :chat-id         chat-id
+                    :jail-id         (or bot chat-id)
+                    :data-type       :preview
+                    :event-after     (fn [command-message _]
```

Comment:
it's ok to use `#()` syntax i places where you can see and understand incomming arguments. here i asked @janherich to implement `fn` to understand arguments better